src/hooks/hookResources.ts

```diff
@@ -463,3 +490,67 @@ export function handleHook(
     res.json(buildErrorCard('Unknown Error'));
   }
 }
+
+export function createQuestionnaireSuggestion(
+  card: Card,
+  requirement: Requirement,
+  patient: Patient
+) {
+  if (requirement.appContext && requirement.appContext.includes('=')) {
+    const qArr = requirement.appContext.split('='); // break up into parts
+    let qUrl = null;
+    for (let i = 0; i < qArr.length; i++) {
+      if (qArr[i].toLowerCase() === 'questionnaire') {
+        if (i + 1 < qArr.length) {
+          // not at end of array
+          qUrl = qArr[i + 1];
+        }
+      }
+    }
+    if (qUrl) {
+      const action: Action = {
+        type: 'create',
+        description: `Create task for "completion of ${requirement.name} Questionnaire`,
+        resource: createQuestionnaireCompletionTask(requirement.name, qUrl, patient)
+      };
+      const suggestion: Suggestion = {
+        label: `Add "Completion of ${requirement.name} Questionnaire" to task list`,
+        actions: [action]
```

Comment:
Is there a way to make this such that it can be deselected? Currently you can only "Add" but if the user accidentally clicks then they have to reload the page to reset it.